http/jaxrs/src/test/java/io/quarkus/ts/security/core/MultipartResourceIT.java

```diff
@@ -0,0 +1,111 @@
+package io.quarkus.ts.security.core;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Random;
+
+import javax.ws.rs.core.MediaType;
+
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+import io.restassured.builder.MultiPartSpecBuilder;
+import io.restassured.http.ContentType;
+import io.restassured.response.ValidatableResponse;
+import io.restassured.specification.MultiPartSpecification;
+
+@QuarkusScenario
+public class MultipartResourceIT {
```

Comment:
Do we want to add corresponding OpenShift scenarios as well at this moment, or is it planned as a future step?